docs/feature-engineering.rst

```diff
@@ -37,6 +37,17 @@ The statistics available include:
 
 Putting the operation and the statistic together, you can see some examples. To get stddev of classic_idf, you would write :code:`stddev_classic_idf`. To get the minimum total term frequency, you'd write :code:`min_raw_ttf`.
 
+Term position statistics
+
+The :code:`type` parameter can be prepended with the operation to be performed across term position for the statistic :code:`min`, :code:`max` and :code:`avg`.
+For any of the cases, 0 will be returned if there isn't any occurrence of the terms in the document.
+
+The statistics available include, e.g. using the query "dance monkey" we have:
+
+- :code:`min_raw_tp` -- return the minimal occurrence, i.e. the first one, of any term on the query. So if dance occurs at positions [2, 5 ,9], and monkey occurs at positions [1, 4] in a text in the same document, the minimal is 1.
```

Comment:
It doesn't seem like I can edit your PR directly, can you change these lines to look like the following:

```
- :code:`min_raw_tp` -- return the minimum occurrence, i.e. the first one, of any term on the query. So if dance occurs at positions [2, 5 ,9], and monkey occurs at positions [1, 4] in a text in the same document, the minimum is 1.
 - :code:`max_raw_tp` -- return the maximum occurrence, i.e. the last one, of any term on the query. So if dance occurs at positions [2, 5 ,9] and monkey occurs at positions [1, 4] in a text in the same document, the maximum is 9.
```

Changes: `s/minimal/minimum`, and the description for max was using min in its definition.

If we can get this updated I think we're ready to merge.  Thanks for the updates!